### PR TITLE
Add COVID-19 report

### PR DESCRIPTION
**Pull request by leobcgov**
# Fixes or Implements  [AB#9605](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9605) & [AB#9606](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9606)

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Description
- Add COVID-19 report page and selection.
image
image

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [X] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES

### Browsers Tested

* [X] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/components/report/covid19.vue [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+            });
+    }
+}
+</script>
+
+<template>
+    <div>
+        <LoadingComponent :is-loading="isLoading"></LoadingComponent>
+        <div ref="report">
+            <section class="pdf-item">
```
Comment (tiagograf-AOT) [inline Apps/WebClient/src/ClientApp/src/components/report/covid19.vue:151]: I pushed a change that makes the header conditional, look at the changes in the PR below and apply those to this report too!
https://github.com/bcgov/healthgateway/pull/1425/files

File: Apps/WebClient/src/ClientApp/src/views/reports.vue [hunk 2]
```diff
@@ -141,6 +154,21 @@ export default class ReportsView extends Vue {
                             </div>
                         </b-col>
                     </div>
+                    <div
+                        v-else-if="reportType == 'COVID-19'"
+                        data-testid="covid19ReportSample"
+                    >
+                        <b-col>
```
Comment (tiagograf-AOT) [inline Apps/WebClient/src/ClientApp/src/views/reports.vue:161]: this changed a bit too, look at the same PR https://github.com/bcgov/healthgateway/pull/1425/files

**Review by tiagograf-AOT — CHANGES_REQUESTED** [review]
few comments about a new code i just pushed to dev!

File: Apps/WebClient/src/ClientApp/src/components/report/covid19.vue (hunk 1 above)
Comment (leobcgov) [reply Apps/WebClient/src/ClientApp/src/components/report/covid19.vue:154]: updated

File: Apps/WebClient/src/ClientApp/src/views/reports.vue (hunk 2 above)
Comment (leobcgov) [reply Apps/WebClient/src/ClientApp/src/views/reports.vue]: updated

**Review by tiagograf-AOT — APPROVED** [review]

**Review by sslaws — APPROVED** [review]

**Pull request merged**